lib/packagekit-glib2/pk-client.c

```diff
@@ -1083,8 +1085,12 @@ pk_client_signal_cb (GDBusProxy *proxy,
 			       &tmp_uint,
 			       &tmp_str[1],
 			       &tmp_str[2]);
+		/* The 'info' and 'update-severity' are encoded in the single value */
+		tmp_uint2 = tmp_uint & 0xFFFF;
+		tmp_uint3 = (tmp_uint >> 16) & 0xFFFF;
```

Comment:
note to other reviewers: this was acked by me, as i really don't want to break DBus API.